t/10-tests_overview.t

```diff
@@ -65,6 +65,17 @@ like(get_summary, qr/Overall Summary of opensuse 13\.1 build 0091/i, 'specifying
 $form = {distri => 'opensuse', version => '13.1', build => '0091', groupid => 1001};
 $t->get_ok('/tests/overview' => form => $form)->status_is(200);
 like(get_summary, qr/Overall Summary of opensuse build 0091/i, 'specifying groupid parameter');
+$form = {
+    distri  => '<img src="distri">',
+    version => ['<img src="version1">', '<img src="version2">'],
+    build   => '<img src="build">'
+};
+$t->get_ok('/tests/overview' => form => $form)->status_is(200);
+like(
+    get_summary,
+    qr/Overall Summary of <img src="distri"> <img src="version1">\/<img src="version2"> build <img src="build">/i,
+    'escaping works'
+);
```

Comment:
What escaping? That doesn't look escaped to me.

Reply:
But `get_summary` uses `$t->tx->res->dom->at('#summary')->all_text`. I would have expected that `all_text` would only return these tags **as text** if the encoding worked.

Reply:
I've just tested:

```
print(Mojo::DOM->new->parse("<div>foo <img src=\"bar\"></div>")->at('div')->all_text);
```
This returns *only* foo because the `img` tag is *not* escaped. So this test should work.

And besides, I tested this manually by viewing the source code.

Reply:
Then i don't like the test, it should make sure the actual output the user sees is escaped. It's a security feature after all.

Reply:
This test can be there in addition to something checking the actual encoded entities. But i think the encoded entities should be checked explicitly.

Reply:
I changed the test to look for excaped `img` tags and to be sure also for the absence of unescaped tags.